includes/multi-currency/wc-payments-multi-currency.php

```diff
@@ -17,6 +17,13 @@ function WC_Payments_Multi_Currency() { // phpcs:ignore WordPress.NamingConventi
 }
 
 add_action( 'plugins_loaded', 'WC_Payments_Multi_Currency', 12 );
+add_action(
+	'plugins_loaded',
+	function() {
+		WC_Payments_Multi_Currency()->init_hooks();
+	},
+	12
+);
```

Comment:
Added the call to `init_hooks` via an action here due to that's how MC is initialized in the first place. The action is added after the one that initializes MC with the same priority so that it is done immediately after.